source/examples/rapids-coiled-cudf/notebook.ipynb

```diff
@@ -0,0 +1,2426 @@
+{
```

Comment:
<p>Could we switch the wording around here to be a little more verbose. Something like "This notebook was designed to be used on Coiled Notebooks to demonstrate how to quickly and easily leverage cloud GPU resources that are proximate to the data you are processing".</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/deployment/pull/547/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>